packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -579,6 +606,14 @@ export type CleanNotifications = {
   notificationsById: Record<string, InboxNotificationData>;
 };
 
+export type CleanThreadSubscriptions = {
+  /**
+   * Thread subscriptions by key (kind + subject ID).
+   * e.g. `thread:${string}`, `$custom:${string}`, etc
+   */
+  subscriptions: SubscriptionsByKey;
+} & CleanNotifications;
```

Comment:
I really dislike the `&` operator in TypeScript for how opaque it makes type definitions. It barely stands out from looking at this type definition here that apparently on CleanThreadSubscriptions there also exist `sortedNotifications` and `notificationsById` fields. Would you consider duplicating these field definitions here, for readability/transparency?

Reply:
Good point, I can also use this as an opportunity to comment on why do we even have notifications here